locales/en-US/server.ftl

```diff
@@ -227,10 +227,10 @@ textSizeSmall = Small
 textSizeMedium = Medium
 textSizeLarge = Large
 # Confirm and Cancel button title shown when using text tool
-annotationTextConfirmButton =
-    .title = Confirm
-annotationTextCancelButton =
-    .title = Cancel
+annotationTextConfirmButton = Confirm
```

Comment:
Unfortunately both these strings need a new ID.

Reply:
Ah good to know, @flodolo will the updated ID work